tests/dialects/test_singlestore.py

```diff
@@ -18,3 +18,23 @@ def test_basic(self):
             "SELECT `data`.`id` AS `my_id` FROM `data` AS `data` WHERE `data`.`my_id` = 1 GROUP BY `data`.`my_id` HAVING `data`.`id` = 1",
             ast.sql(dialect=self.dialect),
         )
+
+    def test_time_formatting(self):
+        self.validate_identity(
+            "SELECT TO_DATE('03/01/2019', 'MM/DD/YYYY') AS result",
+            write_sql="SELECT STR_TO_DATE('03/01/2019', '%m/%d/%Y') AS result",
```

Comment:
Nit: let's remove `write_sql` and make the argument positional to match the existing style.
```suggestion
            "SELECT STR_TO_DATE('03/01/2019', '%m/%d/%Y') AS result",
```